flags.go

```diff
@@ -0,0 +1,57 @@
+package fangs
+
+import (
+	"reflect"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+)
+
+type AddsFlags interface {
+	AddFlags(flags *pflag.FlagSet)
+}
+
+func AddCommandFlags(cmd *cobra.Command, structs ...any) {
```

Comment:
this helper could probably be deleted since `Flags()` and `PersistentFlags()` are both the same type (that is, instead of adding another helper to mirror all things to pull off of a cmd, allow the caller to do it)